index.html

```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
             </button>
         </div>
     </section>
-    <div class="myrecentwork">
-        <h1>My Recent Works</h1>
-    </div>
+    <section>
```

Comment:
Great job @Tekhlay 🤝

- [ ] As stated in the project requirements, kindly align the elements of the `My works section` with `display: grid`.